docs/Submodules.md

```diff
@@ -0,0 +1,53 @@
+Submodules
+==========
+
+This repository composes ASP.NET Core repos from a collection of many submodules.
+Working with submodules in git introduces some quirks into an otherwise workflow.
+Here are some tips for working with submodules.
+
+## Cloning
+
+By default, submodules will not be present. Use `--recursive` to clone all submodules.
+
+    git clone https://github.com/aspnet/Universe.git --recursive
+
+If you have already cloned, run this to initialize all submodules.
+
+    git submodule update --init
+
+## Pulling updates
+
+When you execute `git pull`, submodules do not automatically snap to the version
+used in new commits of the master repo. Update all modules to match the master repo
```

Comment:
Is "master repo" the correct term? I thought it was "parent repo". Just to avoid confusion with "master branch".

Reply:
I'm not really sure, Parent repo sounds better so I'll update .